bevy_lint/action.yml

```diff
@@ -0,0 +1,32 @@
+name: Install `bevy_lint`
+author: BD103
+description: |
+  Installs the Bevy linter and its required toolchain.
+
+  This action currently uses Rustup to install the required toolchain components, and it uses
+  `cargo install` to install the linter itself. The linter will be installed to `~/.cargo/bin` and
+  will be available through the `PATH` by default.
+  
+  Note that this currently will override the default Rustup toolchain to nightly. You can fix this
+  by calling `rustup default` after this action is run. (See
+  <https://github.com/dtolnay/rust-toolchain/pull/144>.)
```

Comment:
As stated, this behavior can be fixed with https://github.com/dtolnay/rust-toolchain/pull/144.

Reply:
That PR has been closed as unwanted, so I'm going to remove the note. Oh well 🥲